spikeinterface/widgets/sortingview/unit_waveforms.py

```diff
@@ -10,7 +10,8 @@ def do_plot(self, data_plot, **backend_kwargs):
         backend_kwargs = self.update_backend_kwargs(**backend_kwargs)
         
         # ensure serializable for sortingview
-        unit_ids, channel_ids, sparsity = self.make_serializable(dp.unit_ids, dp.channel_ids, dp.all_templates)
+        #unit_ids, channel_ids, sparsity = self.make_serializable(dp.unit_ids, dp.channel_ids, dp.all_templates)
+        unit_ids, channel_ids = self.make_serializable(dp.unit_ids, dp.channel_ids)
```

Comment:
Why? This could fail if the channel ID in the sparsity are int64! Safer to make it serializable

Reply:
OK you put all_templates in the input.
I did not understood why.
so the sparsity was wrong.

Reply:
i will change

Reply:
but sparsity was not sended to the json. no ?
whatever I put it back

Reply:
No all templates is just the name, if you want change it back. They are only for the unit ids